### PR TITLE
Changed the team page

### PR DESCRIPTION
**Pull request by cassiecl**

### CODE REVIEW

**Comment by cassiecl** [comment]
Here's my comment :100:

File: templates/base.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
       <meta name="description" content="">
       <meta name="author" content="">
       <link rel="icon" href="../../favicon.ico">
-
+      <link href='http://fonts.googleapis.com/css?family=Crafty+Girls' rel='stylesheet' type='text/css'>
```
Comment (cassiecl) [inline templates/base.html]: Makes the best font on the Internet

**Comment by masnes** [comment]
:+1:

**Comment by thejinxters** [comment]
I merged master in, and I am going to merge it.

**Pull request merged**